README.md

```diff
@@ -1,10 +1,58 @@
-[![appveyor](https://img.shields.io/appveyor/ci/roji/npgsql-entityframeworkcore-postgresql/dev.svg?label=appveyor)](https://ci.appveyor.com/project/roji/npgsql-entityframeworkcore-postgresql) [![travis](https://img.shields.io/travis/npgsql/npgsql.svg?label=travis)](https://travis-ci.org/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL) [![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
+[![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
 
 Npgsql.EntityFrameworkCore.PostgreSQL is an Entity Framework Core provider built on top of
 [Npgsql](https://github.com/npgsql/npgsql). It allows you to use the EF Core O/RM with
 PostgreSQL.
 
 Documentation is available on [the Npgsql site](http://www.npgsql.org/efcore/index.html).
+
+# Build Status
+
+| Branch                               | Build Status                                     | Stable                                 | Next Patch                                  | Unstable                                                    |
+|:-------------------------------------|:-------------------------------------------------|:---------------------------------------|:--------------------------------------------|:------------------------------------------------------------|
+| [`dev`][tree-dev]                    | [![status][build-dev]][ci-dev]                   | [![stable][nuget-vpre]][nuget-npgsql]  |                                             | [![unstable][npgsql-unstable-vpre]][myget-npgsql-unstable]  |
+| [`master`][tree-master]              | [![status][build-master]][ci-master]             | [![stable][nuget-v]][nuget-npgsql]     | [![next patch][npgsql-vpre]][myget-npgsql]  |                                                             |
+| [`hotfix/2.2.6`][tree-hotfix/2.2.*]  | [![status][build-hotfix/2.2.*]][ci-hotfix/2.2.*] | [![stable][nuget-2.2.*]][nuget-npgsql] | [![next patch][npgsql-2.2.*]][myget-npgsql] | [![unstable][npgsql-unstable-2.2.*]][myget-npgsql-unstable] |
+| [`hotfix/2.1.11`][tree-hotfix/2.1.*] | [![status][build-hotfix/2.1.*]][ci-hotfix/2.1.*] | [![stable][nuget-2.1.*]][nuget-npgsql] | [![next patch][npgsql-2.1.*]][myget-npgsql] | [![unstable][npgsql-unstable-2.1.*]][myget-npgsql-unstable] |
+
+
+[* Package Feeds]: <>
+[nuget-npgsql]:          https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL
+[myget-npgsql]:          https://www.myget.org/feed/npgsql/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL
```

Comment:
Azure Devops also has a nuget feature right? We could consider moving there (although totally out of scope for this PR of course).

Reply:
Yes, planning to tackle that along with the formal release pipeline ahead of 3.0.